src/window.rs

```diff
@@ -169,6 +184,18 @@ impl<'a, F, W> Iterator for Windower<'a, F, W>
     }
 }
 
+impl<S, W> Iterator for Windowed<S, W>
```

Comment:
Yeah this is a good point. I have a couple uncertainties with the window API atm, maybe I'll open another issue.